EXTCAP.md

```diff
@@ -0,0 +1,180 @@
+# Spinel Extcap Reference
+The Spinel Extcap provides a more user-friendly way to use OpenThread Sniffer. Spinel 
+Extcap is primarily targeted for integrating OpenThread Sniffer with Wiresahrk,  and 
+is suitable for user to use OpenThread Sniffer by Wireshark GUI.
+
+## System Requirements
+
+The tool has been tested on the following platforms:
+
+| Platforms  | Version          |
+|------------|------------------|
+| Ubuntu     | 14.04 Trusty     |
+| Mac OS     | 10.11 El Capitan |
+| Windows 10 | 1803             |
+
+| Language  | Version          |
+|-----------|------------------|
+| Python    | 2.7.10           |
+
+| Software  | Version          |
+|-----------|------------------|
+| Wireshark | 3.0.3            |
+
+### Package Installation
+#### Automatic installation from source
+```
+git clone https://github.com/openthread/pyspinel
+cd pyspinel
+sudo python setup.py install 
+```
+#### Automatic install from PYPI (pending for new release package)
+```
+pip install pyspinel
+```
+#### Manual installation(for pyspinel 1.0.0a3 and before)
+**1. Install pyspinel package**
+``` 
+pip install pyspinel 
+```
+**2. Install extcap script on Wireshark**
+
+To find the correct installation path of the extcap utility on any system please open Wireshark:
+```
+"Help" -> "About Wireshark" -> "Folders" -> "Extcap path"
+```
+Copy the provided ```extcap_ot.py```, ```extcap_ot.bat``` and ```sniffer.py``` to the extcap directory.
```

Comment:
Nit: Inline code font can just be single backticks like

<div>`extcap_ot.py`</div>